### PR TITLE
MTM-61598 DPT widget documentation update

### PR DESCRIPTION
**Pull request by luja-c8y**

### CODE REVIEW

**Review by BeateRixen — APPROVED** [review]

**Comment by luja-c8y** [comment]
@BeateRixen along with data points table migration, we no longer support "dashboard time range" for this widget in a way shown on a screenshot so I had to update a screenshot and remove its name.
DPT widget will be connected to "global auto refresh context", adn feature allowing that is already merged. Documentation will be updated in this task: https://cumulocity.atlassian.net/browse/MTM-61955

**Comment by BeateRixen** [comment]
> @BeateRixen along with data points table migration, we no longer support "dashboard time range" for this widget in a way shown on a screenshot so I had to update a screenshot and remove its name. DPT widget will be connected to "global auto refresh context", adn feature allowing that is already merged. Documentation will be updated in this task: https://cumulocity.atlassian.net/browse/MTM-61955

@luja-c8y Looks good to me. Please have it approved by someone from your team.

File: static/images/users-guide/cockpit/cockpit-datapointtable.png
Comment (jdre-c8y) [inline static/images/users-guide/cockpit/cockpit-datapointtable.png:1]: I suggest to make screenshot wider so headers (c8y_Acceleratio...) are not trimmed

**Review by jdre-c8y — APPROVED** [review]

File: static/images/users-guide/cockpit/cockpit-datapointtable.png
Comment (luja-c8y) [reply static/images/users-guide/cockpit/cockpit-datapointtable.png:1]: Done

**Pull request merged**